syntax/forth.vim

```diff
@@ -1,10 +1,10 @@
 " Vim syntax file
-" Language:    FORTH
-" Current Maintainer:  Johan Kotlinski <kotlinski@gmail.com>
-" Previous Maintainer:  Christian V. J. Br�ssow <cvjb@cvjb.de>
-" Last Change: 2023-01-12
-" Filenames:   *.fs,*.ft
-" URL:	       https://github.com/jkotlinski/forth.vim
+" Language:		FORTH
+" Current Maintainer:	Johan Kotlinski <kotlinski@gmail.com>
+" Previous Maintainer:	Christian V. J. Br�ssow <cvjb@cvjb.de>
+" Last Change:		2023-01-12
+" Filenames:		*.fs,*.ft
```

Comment:
I suppose the filenames should be updated to "*.f *.fs *.ft *.fth *.4th"

Reply:
Done.

I'm not sure the line is necessary but it certainly doesn't hurt.